lib/parse.js

```diff
@@ -1320,11 +1320,17 @@ function parse($TEXT, options) {
             var type = start.type;
             var name = as_property_name();
             if (type == "name" && !is("punc", ":")) {
+                var createAccessor = function() {
```

Comment:
I have tried your solution once again and I see its error is different. It outputs no parsing error, only the test is not passed, with assertion:

> AssertionError: 22 == 12